src/others/components/LanguageSelector.tsx

```diff
@@ -21,16 +20,20 @@ const Flag = ({ lang, className }: { lang: AvailableLang; className?: string })
   );
 };
 
-function useOutsideClick(ref: React.RefObject<HTMLElement>, onClick: () => void) {
+function useOutsideClick(
+  ref: React.RefObject<HTMLElement>,
+  onClick: () => void,
+  capture?: boolean
```

Comment:
I think maybe you don't need this extra variable, or you should be passing in `{ capture }` below instead of `{ capture: true }`